### DISCUSSION TITLE
Where report issues on joomla-website

### DISCUSSION THREAD

**Discussion by ghost** [joomla/joomla-cms#37684] [Q&A] [answered]
Dear Community

It looks like reports about [jorg] don't work. Example: https://github.com/joomla/joomla-websites/issues/1728 (opened 16 february 2022)

The related team https://volunteers.joomla.org/teams/community-sites-team is empty, can anyone give me a hint where to report?

**Comment by brianteeman** [comment, marked as answer]
you reported it in the correct place

**Reply by ghost** [reply]
Thanks @brianteeman for the answer. Closing and hope there is a fallback for the empty team.
